requirements.txt

```diff
@@ -1,30 +1,51 @@
-#
-# Requirements for wger for production
-#
-
-Django>=1.9,<1.10
-django-recaptcha
+alabaster==0.7.11
+Babel==2.6.0
+bleach==2.1.3
+certifi==2018.4.16
+chardet==3.0.4
+coverage==4.5.1
+coveralls==1.3.0
```

Comment:
Am curious as to whether these dependencies are required in the production environment. Most of the new ones seem to be for development purposes only. Maybe they should be in requirements-devel.txt file.